ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -0,0 +1,344 @@
+      shouldShowSummary: false,
+      hideTimer: true
+    };
+
+    this.setState({
+      scaffolding,
+      gameSession: {
+        email,
+        drawResponseMode: this.drawResponseMode,
+        drawStudentCard: this.drawStudentCard,
+        drawResponsePrompt: this.drawResponsePrompt,
+        sessionId: uuid.v4(),
+        questions: withStudents(dansonQuestions),
+        questionsAnswered: 0,
+        msResponseTimes: [],
+        audioResponses: []
+      },
+    });
+  },
+
+  render() {
+    return (
+      <ResponsiveFrame>
+        <div>
+          <NavigationAppBar
+            title="Teacher Moments"
+            prependMenuItems={
+              <MenuItem
+                onTouchTap={this.resetExperience}
+                leftIcon={<RefreshIcon />}
+                primaryText="Restart game" />
+            }
+          />
+          {this.renderMainScreen()}
+        </div>
+      </ResponsiveFrame>
+    );
+  },
+
+  renderMainScreen() {
+    // configure the game
+    const {gameSession} = this.state;
+    const hasStarted = gameSession !== null;
+    if (!hasStarted) return this.renderInstructions();
+
+    // all done!
+    const questionsAnswered = hasStarted ? gameSession.questionsAnswered : 0;
+    const questions = hasStarted ? gameSession.questions : [];
+    const sessionLength = hasStarted ? questions.length : 0;
+    if (questionsAnswered >= sessionLength) return this.renderDone();
+
+    // text-messaging prototype
+    if (_.has(this.props.query, 'mobilePrototype')) return this.renderMobilePrototype();
+
+    // question screen
+    return this.renderPopupQuestion();
+  },
+
+  renderDone() {
+    const audioResponses = this.state.gameSession.audioResponses;
+    const truncatedAudioResponses = [];
+    const maxCharPerLine = 120;
+    for(var i = 0; i < audioResponses.length; i++) {
+      var obj = {};
+      obj.audioUrl = audioResponses[i].audioUrl.uploadedUrl;
+      obj.questionText = audioResponses[i].question.text.length < maxCharPerLine ? audioResponses[i].question.text : audioResponses[i].question.text.substring(0, maxCharPerLine) + ' ...';
+      truncatedAudioResponses.push(obj);
+    }
+    return (
+      <div className="done">
+        <VelocityTransitionGroup enter={{animation: "slideDown"}} leave={{animation: "slideUp"}} runOnMount={true}>
+          <div style={styles.doneTitle}>
+            <p style={styles.paragraph}>You've finished the simulation.</p>
+            <p style={styles.paragraph}><strong>Do not close this page</strong>. You will need it for the reflection.</p>
+            <p style={styles.paragraph}>Please return to the form for the post-simulation reflection.</p>
+          </div>
+          <p style={styles.summaryTitle}>Summary</p>
+          <Divider />
+          {truncatedAudioResponses.map((obj, i) =>
+            <div key={i} style ={_.merge(styles.instructions)}>
+              <p style={styles.paragraph}>{obj.questionText}</p>
```

Comment:
It comes out in line 193. The truncatedAudioResponses array exists solely for the purpose of adding ellipsis. It seems like a lot of code for a simple functionality. But that was my quickest hack.